### DISCUSSION TITLE
Re:  Add a date modified column to the trust list view #61

### DISCUSSION THREAD

**Discussion by tparchambault** [ctc-oss/fapolicy-analyzer#229] [Developers]
I'm currently working on adding another field to display a file's inode mtime value, i.e. last mod time. Currently I intend to steal the needed UI square footage from the absolute path field area. However...

On first design iteration, I'm not convinced that the mtime details should be in the file list object itself given the limited UI real-estate we have available there, and given that it is in the 'On File System' details for both the ancillary and system databases.

We can still sort by mtime, and potentially make a column optionally visible to let the user decide how they want their UI area allocated.

**Comment by jw3** [comment]
Some thoughts

1. Given that last modified is a significant contributor to discrepancy, it seems useful to see it as well as optionally sort by it.
2. It might be interesting to render the timestamp in timeago format, which would save space.
3. Things get extra cramped when you have an edit queued up because it adds that extra column.
     - If you consolidate the edit rendering into one of the other columns using an icon or a textual badge it would save some space.
4. Configurable columns seem overkill at this time.

**Comment by jw3** [comment]
Closed in
- https://github.com/ctc-oss/fapolicy-analyzer/pull/236
